acceptance/indy-cli-batches/AS-03-01-identity-owner-anyone-can-write.batch

```diff
@@ -1,133 +1,133 @@
-# #6 AS IdentityOwner CREATE Endorser (FAIL)
-did use LBbKEeczA9iL21p4Kgxcuf
-- ledger nym did=2b6xTx2HniDU77nxHm6zWB role=ENDORSER verkey=~HF34ymwfTJngb8zFDmCyvX
-- ledger get-nym did=2b6xTx2HniDU77nxHm6zWB
-# #7.1 AS IdentityOwner CREATE IdentityOwner (PASS with ANYONE_CAN_WRITE=True)
-did use LBbKEeczA9iL21p4Kgxcuf
-ledger nym did=Fk9ENxnz1ztDgdivQBJTCZ verkey=~NT9ANFeDhCLKDg5PNKZXKK
-- ledger get-nym did=Fk9ENxnz1ztDgdivQBJTCZ
-# #8 AS IdentityOwner CREATE NetworkMonitor (FAIL)
-did use LBbKEeczA9iL21p4Kgxcuf
-- ledger nym did=Nd4dUF85aa1JmDFScLfk7o role=NETWORK_MONITOR verkey=~5aJLqtphR3tqymWUwg98Sz
-# #9.1 AS IdentityOwner REMOVE NetworkMonitor (FAIL)
-did use LBbKEeczA9iL21p4Kgxcuf
-- ledger nym did=GSoagH52cX69hnn7npUTWy role=
-# #9.2 CHECK NetworkMonitor IS VALID
-did use GSoagH52cX69hnn7npUTWy
-ledger get-validator-info nodes=Node1
-# #10 check that can't add roles back when don't have necessary permissions
-did use LBbKEeczA9iL21p4Kgxcuf
-- ledger nym did=Q8uxmCGdXgLHHtaTwYtVJG role=ENDORSER
-- ledger nym did=Q8uxmCGdXgLHHtaTwYtVJG role=STEWARD
-- ledger nym did=Q8uxmCGdXgLHHtaTwYtVJG role=TRUSTEE
-- ledger nym did=CjPA39BhAVG5d3mV2ZEAYE role=NETWORK_MONITOR
-### Additional cases ###
-did use V4SGRU86Z58d6TV7PBUe6f
-did new seed=RandomAttribOwner000000000000003
-ledger nym did=UffJCJngTXc1o84dQ7aEUb verkey=~J4HtnGz2wW2nE7VuMeg39M
-did new seed=RandomUserForRotateKey0000000003
-ledger nym did=X3zYajU7gbz9Pu8k6E7Ppf verkey=~3ZDo6g4ZDRKGauKrR452xU
-# Only owner can edit existing NYMs
-did use X3zYajU7gbz9Pu8k6E7Ppf
-did rotate-key seed=RandomUserForRotateKey0NewKey003
-# Not owner can't edit existing NYMs (FAIL)
-did use UffJCJngTXc1o84dQ7aEUb
-- ledger nym did=X3zYajU7gbz9Pu8k6E7Ppf verkey=~3ZDo6g4ZDRKGauKrR452xU
-# Only owners can create ATTRIBs
-did use X3zYajU7gbz9Pu8k6E7Ppf
-ledger attrib did=X3zYajU7gbz9Pu8k6E7Ppf raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Only owners can edit ATTRIBs
-ledger attrib did=X3zYajU7gbz9Pu8k6E7Ppf raw={"endpoint":{"ha":"127.0.0.1:6666"}}
-# Not owner can't create attrib (FAIL)
-did use X3zYajU7gbz9Pu8k6E7Ppf
-- ledger attrib did=UffJCJngTXc1o84dQ7aEUb raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Not owner can't edit attrib (FAIL)
-did use UffJCJngTXc1o84dQ7aEUb
-- ledger attrib did=X3zYajU7gbz9Pu8k6E7Ppf raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Identity Owner can create SCHEMA: (PASS with ANYONE_CAN_WRITE=True)
-ledger schema name=IdentityOwnerSchema version=1.0 attr_names=name,age
-# Identity Owner can create CLAIM_DEF: (PASS with ANYONE_CAN_WRITE=True, but will fail because of wrong schema id)
-- ledger cred-def schema_id=1 signature_type=CL primary={"n":"1","s":"2","rms":"3","r":{"age":"4","name":"5"},"rctxt":"6","z":"7"} tag=3
-# Identity Owner can't RESTART POOL
-did use LBbKEeczA9iL21p4Kgxcuf
-- ledger pool-restart action=start datetime=2020-01-25T12:49:05.258870+00:00 nodes=Node4
-# Identity Owner can't get validator-info
-did use LBbKEeczA9iL21p4Kgxcuf
+# setup environment
```

Comment:
On a clean clone of the repo: 
```
root@73570c5a4ed8:/indy-node# git diff -w
warning: CRLF will be replaced by LF in acceptance/indy-cli-batches/AS-03-01-identity-owner-anyone-can-write.batch.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in acceptance/indy-cli-batches/AS-03-01-steward.batch.
The file will have its original line endings in your working directory.
```

Due to 

> # Set the default behavior, in case people don't have core.autocrlf set.
> * text=auto

in .gitattributes.

If LF is the desired line ending, these files have to change either in this PR or another one.